ivadomed/training.py

```diff
@@ -101,6 +102,7 @@ def train(model_params, dataset_train, dataset_val, training_params, log_directo
     best_training_dice, best_training_loss = float("inf"), float("inf")
     best_validation_loss, best_validation_dice = float("inf"), float("inf")
     patience_count = 0
+    begin_time = time.time()
 
     # EPOCH LOOP
     for epoch in tqdm(range(1, num_epochs + 1), desc="Training"):
```

Comment:
Just curious: have you look at whether `tqdm` could do this job?
Otherwise, all good, good to merge.

Reply:
Yes I checked. I haven't found an answer easily so I thought this was quicker

Reply:
Makes sense :-)